libraries/Adapters/Microsoft.Bot.Builder.Adapters.WeChat/Schema/JsonResult/WeChatJsonResult.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Adapters.WeChat.Schema.JsonResult
+{
+    public class WeChatJsonResult
+    {
+        /// <summary>
+        /// Gets or sets ErrorCode.
+        /// </summary>
+        /// <value>
+        /// Error code from WeChat, default is 0.
+        /// </value>
+        [JsonProperty("errcode")]
+        public int ErrorCode { get; set; }
+
+        /// <summary>
+        /// Gets or sets ErrorMessage.
+        /// </summary>
+        /// <value>
+        /// Error message from WeChat, default is 'ok'.
+        /// </value>
+        [JsonProperty("errmsg")]
+        public virtual string ErrorMessage { get; set; }
+
+        public override string ToString() => string.Format("WeChatJsonResult：{{ErrorCode:'{0}',ErrorCode_name:'{1}',ErrorMessage:'{2}'}}", ErrorCode, ErrorCode.ToString(), ErrorMessage);
```

Comment:
Pass cultureInfo to string.format (not sure which one is right here InvariantCulture or UICulture).

Check this article to find out more in case it helps you decide:
https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1305-specify-iformatprovider?view=vs-2019

Reply:
I think I should use invariant culture here.